packages/xo-web/src/xo-app/pool/tab-advanced.js

```diff
@@ -458,13 +468,22 @@ export default class TabAdvanced extends Component {
                         value={backupNetwork}
                         xoType='network'
                       >
-                        {backupNetwork !== undefined ? <Network id={backupNetwork.id} /> : _('noValue')}
+                        {pool.otherConfig['xo:backupNetwork'] === undefined ? (
+                          _('noValue')
+                        ) : backupNetwork !== undefined ? (
+                          <Network id={backupNetwork.id} />
+                        ) : (
+                          <span className='text-danger'>
+                            {_('updateMissingNetwork', {
+                              networkID: (
+                                <Copiable onClick={e => e.stopPropagation()}>
```

Comment:
You need to add the `data` prop to explicitly give "what should be copied", otherwise it will copy `children` which is not a string.
`<Copiable data={pool.otherConfig['xo:backupNetwork']}>`